go/tasks/plugins/array/awsbatch/transformer_test.go

```diff
@@ -58,8 +80,8 @@ func TestArrayJobToBatchInput(t *testing.T) {
 			Environment: []*batch.KeyValuePair{
 				{Name: refStr("BATCH_JOB_ARRAY_INDEX_VAR_NAME"), Value: refStr("AWS_BATCH_JOB_ARRAY_INDEX")},
 			},
-			Memory: refInt(700),
-			Vcpus:  refInt(2),
+			Memory: refInt(1074),
```

Comment:
Any specific reason why this and the following constants need to be changed?

Reply:
We used to have AWS Batch defaults (hard coded) that were separate from K8s defaults (driven from config). Now that we use the same code, we no longer need this diversion.